react/features/aeternity/utils.js

```diff
@@ -27,3 +27,16 @@ export function isWalletNameSet(state) {
 
     return false;
 }
+
+/**
+ * Returns true if site is embeded in an iframe.
+ *
+ * @returns {boolean}
+ */
+export function isInIframe() {
+    try {
+        return window.self !== window.top;
+    } catch (e) {
+        return true;
+    }
+}
```

Comment:
I remember that we have the same util in some other file. Can you please leave some one of them?

Reply:
Can you please move https://github.com/aeternity/jitsi-meet/blob/027dcd1c98b8c6e38434a9f99a7f54da1012d4ff/react/features/base/util/inIframe.js#L1-L9 to this file